DONE: System Parameter Partial Maintenance Pass (#170)
makerdao/governance-manual: parameter-index/core/param-system-surplus-buffer.md

In addition, the buffer provides a reserve of DAI which may be used by governance to fund the operations of Core Units and Oracle feeds without requiring the minting of MKR.

## Trade-offs

Increasing the  Maximum System Surplus allows the Maker Protocol to accrue a larger reserve of DAI before burning MKR. This larger reserve provides greater security for the protocol in the event of bad debt.

However, while the buffer is not full FLAP auctions do not take place, and MKR is not burned. This means that Maker Governance is not being rewarded for its efforts during this time.

Additionally, DAI in the System Surplus Buffer is not circulating in the market. This means that holding large amounts of DAI in the System Surplus Buffer will increase upwards pressure on the DAI peg.

> Maintaining too low of a  Maximum System Surplus on the other hand means that FLOP auctions are more likely to take place in the event of bad debt. This makes it more likely that the supply of MKR will increase and dilute the value of current MKR Holders.

Reviewer: ```suggestion
On the other hand, maintaining too low of a Maximum System Surplus means that FLOP auctions are more likely to take place in the event of bad debt. This makes it more likely that the supply of MKR will increase and dilute the value of current MKR Holders.
```

## Changes

Adjusting the Maximum System Surplus parameter is a manual process that requires an executive vote. Changes to the Maximum System Surplus are subject to the [GSM Pause Delay](param-gsm-pause-delay.md).

**Why increase this parameter?**

> The primary reason for increasing the Maximum System Surplus is if it is judged that more surplus is needed to minimize the risk of MKR minting in the case of a market event leading to bad debt. In general, the larger the total DAI debt, the larger the Maximum System Surplus should be to balance the risk from volatile collateral.

Reviewer: ```suggestion
The primary reason for increasing the Maximum System Surplus is minimizing the risk of MKR minting in the case of a market event leading to bad debt. In general, the larger the total DAI debt, the larger the Maximum System Surplus should be to balance the risk from volatile collateral.
```

Another reason to increase the Maximum System Surplus is if for whatever reason it is beneficial to prevent the burning of MKR for a certain time.

**Why decrease this parameter?**

> The primary reason for decreasing the Maximum System Surplus when the buffer is full would be if Governance wished to use the DAI currently in the buffer to buy and burn MKR. Alternatively, if the buffer was not full, Governance may choose to decrease the buffer to restart the continuous MKR burn arising from the buffer overflow.

Reviewer: ```suggestion
The primary reason for decreasing the Maximum System Surplus when the buffer is full is to use the DAI currently in the buffer to buy and burn MKR, if governance so wishes. Alternatively, if the buffer was not full, Governance may choose to decrease the buffer to restart the continuous MKR burn arising from the buffer overflow.
```

Reviewer: Not happy with the rewording, but I think it's too colloquial as it is now
